docs/guide/request_lifecycle.md

```diff
@@ -0,0 +1,155 @@
+[TODO:
+- Replace internal absolute link with relative links: 'https://doc.ibexa.co/en/latest/guide/' → ''
+- Permission Control
+]
+
+# Request Lifecycle: From Request to Response
+
+## Intro: HTTP Request entering the architecture
+
+When entering the architecture, the HTTP request can be handled by several component like a firewall, a load balancer, a reverse-proxy, etc. before arriving on the web server itself.
+
+For an overview of what happens on reverse proxy like Varnish or Fastly, see [Context-aware HTTP cache / Request lifecycle](https://doc.ibexa.co/en/latest/guide/cache/context_aware_cache/#request-lifecycle).
+
+When arriving at a Web server, the request is filtered by Apache Virtual Host, Nginx Server Blocks or equivalent. This mainly (and shortly) separates requests of static resources from requests to the application.
+
+As Ibexa DXP is a Symfony application, everything starts like in Symfony.
+- https://symfony.com/doc/current/introduction/http_fundamentals.html#requests-and-responses-in-http
+- https://symfony.com/doc/current/introduction/http_fundamentals.html#the-symfony-application-flow
+
+If the HTTP request is to be treated by Ibexa DXP, it goes to the [Symfony Front Controller](https://symfony.com/doc/current/configuration/front_controllers_and_kernel.html#the-front-controller) public/index.php
+
+The Front Controller transform the HTTP request into a PHP object `Request` and pass it to Symfony's Kernel to obtain a `Response` object that will be sent back as an HTTP response.
+
+The schemas start with a Request object entering Symfony and Ibexa DXP. No REST or GraphQL HTTP Request here, only regular request from a browser.
+
+
+
+## From Request to Response: Flowcharts
```

Comment:
We use sentence case in headings (only first word, proper names and domain terms from a fixed listed are capitalized):

```suggestion
## From request to response: flowcharts
```

Reply:
@DominikaK 
Before fixing this usage, I must fix the way I use PHP class names. Here, Request and Response refers to PHP class names and should be between backquotes. About class names usage, I have a difficulty with services. I often indicate both there class and their code name. For example “DefaultRouter (`router.default`)”. Maybe I could only use the code names as they're used in config files or when using `bin/console debug:container`.